django/text_extractor/views.py

```diff
@@ -184,12 +184,17 @@ def submit_document(request):
                         },
                     }
                 )
-            formatted_total_cost = display_cad_cost(total_cost)
-            context = {
-                "ocr_docs": completed_documents,
-                "user_request_id": user_request.id,
-                "total_cost": formatted_total_cost,
-            }
+                formatted_total_cost = display_cad_cost(total_cost)
+                context = {
+                    "ocr_docs": completed_documents,
+                    "user_request_id": user_request.id,
+                    "total_cost": formatted_total_cost,
+                }
+            else:
+                context = {
+                    "ocr_docs": completed_documents,
+                    "user_request_id": user_request.id,
+                }
```

Comment:
single costs associated when files are not merged, otherwise total cost is passed.